src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -92,6 +91,8 @@ export const ArtworkSidebarCommercialButtons: React.FC<React.PropsWithChildren<A
   const createOrder = useCreateOrderMutation()
   const createOfferOrder = useCreateOfferOrderMutation()
 
+  const { signals } = useArtworkGridContext()
```

Comment:
Would this populate the data when the `PrimaryLabelLine` component is not rendered with this `ArtworkSidebarCommercialButtons` component?